docs/resources/sms_task.md

```diff
@@ -0,0 +1,162 @@
+  If omitted, it will be obtained from the source server. The [object](#target_server_disks_object)
+  is documented below. Changing this parameter will create a new resource.
+
+* `use_public_ip` - (Optional, Bool, ForceNew) Specifies whether to use a public IP address for migration.
+  The default value is `true`. Changing this parameter will create a new resource.
+
+* `migration_ip` - (Optional, String, ForceNew) Specifies the IP address of the target server.
+  Use the EIP of the target server if the migration network type is Internet.
+  Use the private IP address the target server if the migration network type is Direct Connect or VPN.
+  Changing this parameter will create a new resource.
+
+* `start_target_server` - (Optional, Bool, ForceNew) Specifies whether to start the target server after the migration.
+  The default value is `true`. Changing this parameter will create a new resource.
+
+* `syncing` - (Optional, Bool, ForceNew) - Specifies whether to perform a continuous synchronization after the first replication.
+  The default value `false`. Changing this parameter will create a new resource.
+
+* `action` - (Optional, String) Specifies the operation after the task is created.
+  The value can be **start**, **stop** and **restart**.
+
+* `project_id` - (Optional, String, ForceNew) Specifies the project ID where the target server is located.
+  If omitted, the default project in the region will be used. Changing this parameter will create a new resource.
+
+<a name="target_server_disks_object"></a>
+The `target_server_disks` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the disk name, e.g. "/dev/sda".
+  Changing this parameter will create a new resource.
+
+* `size` - (Required, int, ForceNew) Specifies the volume size in bytes. Changing this parameter will create a new resource.
+
+* `device_type` - (Required, String, ForceNew) Specifies the disk type. The value can be **NORMAL** and **BOOT**.
+  Changing this parameter will create a new resource.
+
+* `disk_id` - (Optional, String, ForceNew) Specifies the disk index, e.g. "0".
+  Changing this parameter will create a new resource.
+
+* `used_size` - (Optional, int, ForceNew) Specifies the used space in bytes. Changing this parameter will create a new resource.
+
+* `physical_volumes` - (Optional, List, ForceNew) Specifies an array of physical volume informations.
+  The [object](#physical_volumes_object) is documented below. Changing this parameter will create a new resource.
+
+<a name="physical_volumes_object"></a>
+The `physical_volumes` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the volume name. In Windows, it indicates the drive letter,
+  and in Linux, it indicates the device ID, e.g. "/dev/sda1".
+  Changing this parameter will create a new resource.
+
+* `size` - (Required, int, ForceNew) Specifies the volume size in bytes. Changing this parameter will create a new resource.
```

Comment:
Not sure what unit we use on console side, but bytes for disk seems a bit inconvinient

Reply:
will do it in a new PR